### PR TITLE
Add eslint@2.x to repository and fix lint errors

### PR DESCRIPTION
**Pull request by kjdelisle**
### Description

Add eslint@2.x and eslint-config-loopback@1.x
Fix lint errors

#### Related Issues
https://github.com/strongloop-internal/scrum-loopback/issues/1330

### CODE REVIEW

File: .eslintrc [hunk 1]
```diff
@@ -0,0 +1,10 @@
+{
+  "extends": "loopback",
+  "rules": {
+    "max-len": ["error", 120, 4, {
```
Comment (loay) [inline .eslintrc]: I think the common usage is 80 characters now.

Comment (kjdelisle) [inline .eslintrc]: Weird. I stole this rule from the mongodb master stream. :S

File: package.json [hunk 2]
```diff
@@ -17,10 +18,12 @@
     "loopback-datasource-juggler": "2.x.x"
```
Comment (0candy) [inline package.json]: You can remove this.

Comment (kjdelisle) [inline package.json]: Not a part of this PR

Comment (kjdelisle) [inline package.json]: I agree it should be done, but in a different one.

**Pull request merged**